about.json

```diff
@@ -16,7 +16,7 @@
         "background": "FBFBFB",
         "header_background": "FFFFFF",
         "header_primary": "333333",
-        "highlight": "F1F2F2",
+        "highlight": "ffff80",
         "danger": "F4595B",
```

Comment:
@lyallstewart for consistency could you please capitalise this (or lowercase all the others). Not fussed which, just need the consistency.